cmd/utils/cmd.go

```diff
@@ -40,7 +41,13 @@ import (
 )
 
 const (
-	importBatchSize = 2500
+	importBatchSize       = 2500
+	freeDiskSpaceWarning  = 1024 * 1024 * 1024
+	freeDiskSpaceCritical = 300 * 1024 * 1024
+)
+
+var (
+	sigtermCh = make(chan os.Signal, 1)
```

Comment:
This doesn't have to be global, if the other changes are made

Reply:
fixed, it's not global anymore